basic-unix/Basic_Unix_Linux.tex

```diff
@@ -43,17 +54,17 @@
 
 \begin{frame}
 \frametitle{General-purpose utilities}
-date, who, uname, echo, passwd, bc
+date, who, uname, echo, passwd, bc, man, time
```

Comment:
Add export. Additionally, maybe we should suggest the use of `python -c "print(...)"` for math to avoid the complexities of floating point math in bc?